### PR TITLE
Add transactions, complex queries, modify and delete data

### PR DESCRIPTION
**Pull request by Gegardus**
In this part of Vet Clinic Database SQL project were:

- Used transactions to update and delete records
- Implemented aggregate functions with GROUP BY to answer analytical questions.

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -1,8 +1,22 @@
...
 ('Devimon', '2017-05-12', 5, TRUE, 11);
+
+-- Day II
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES
+('Charmander', '2020-02-08', 0, FALSE, 11),
+('Plantmon', '2021-11-15', 2, TRUE, 5.7),
+('Squirtle', '1993-04-02', 3, FALSE, 12.13),
+('Angemon', '2005-06-12', 1, TRUE, 45),
```
Comment (Lordkaito) [inline data.sql]: - [x] I know it looks like a mistake, but here, these three animals are supposed to have negative weight, so please change it from 5.7, 12.13, and 45 to -5.7, -12.13, and -45.

**Review by Lordkaito — CHANGES_REQUESTED** [review]
Hi @Gegardus, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: data.sql (hunk 1 above)
Comment (Gegardus) [reply data.sql]: Okay. Although then it looks more like science fiction than a DB :). 
By the way, if so, then Charmander must also have a negative weight.

Comment (Lordkaito) [reply data.sql]: I know what you mean, but you will understand the reason for this soon 😎

**Review by Lordkaito — APPROVED** [review]
Hi @Gegardus, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Gegardus** [comment]
Thank you @Lordkaito

**Pull request merged**